### PR TITLE
Ingest iwclul20 (under new ID format)

### PR DESCRIPTION
**Pull request by mjpost**
This is the first ingestion under the new ID format (#291 #732). Among the updates:

- The `ingest.py` script now works directly from ACLPUB output. No more running `anthologize` in ACLPUB—its scripts were reworked into ingestion. See parallel commit https://github.com/acl-org/ACLPUB/pull/28. There is now one fewer Perl script in the world.
- I removed the `make_nested()` function and CLI script. These were used to convert the old flat XML format which were still produced by the ACLPUB scripts.
- I updated the documentation so the default description is the new format, and moved the description of the old format lower down.

It also includes an ingestion of IWCLUL 2020.

### CODE REVIEW

**Comment by akoehn** [comment]
> It also includes an ingestion of IWCLUL 2020.

I don't see that anywhere.

**Comment by mjpost** [comment]
Oops, forgot to commit that file. Which explains why only my build broke, locally. Will submit soon.

**Comment by mjpost** [comment]
The decade layout adapted quite nicely!

image

All the links worked for me, except for the PDF ones, which will require us to make the new `.htaccess` rules live.

image

**Comment by mjpost** [comment]
Note that most of the changes in this diff are #732. If we merge that first, the diffs here will be clearer.

**Comment by mjpost** [comment]
@acl-org/anthology The top message now includes an updated statement of what this PR includes. This should be ready for merging along with https://github.com/acl-org/ACLPUB/pull/28.

File: bin/anthology/bibtex.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    else:
+        logging.warning("couldn't figure out encoding; using ascii with escapes")
+        bibstring = bibbytes.decode('ascii', 'backslashreplace')
+
+    if bibstring.startswith('\uFEFF'):
+        bibstring = bibstring[1:]  # Unicode BOM
+
+    # for parser in [lambda s: pybtex.database.parse_string(s, 'bibtex'),
+    #                fake_parse]:
+    for parser in [lambda s: pybtex.database.parse_string(s, 'bibtex')]:
```
Comment (akoehn) [inline bin/anthology/bibtex.py]: This line looks weird to me; it iterates through an one element list of parser.  If there are no plans to introduce more parsers, we should simplify that logic.

File: bin/anthology/bibtex.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+# limitations under the License.
+
+import copy
+import re
+import logging
+import pybtex, pybtex.database.input.bibtex
+
+
+def read_bibtex(bibfilename):
+    # Guess encoding. BibTeX is theoretically always in ASCII
```
Comment (akoehn) [inline bin/anthology/bibtex.py]: Maybe a one-line docstring for the method?  "Reads bibtex data from a file into pybtex data structures, guessing the encoding of the file" or something similar.

File: hugo/content/info/ids.md [hunk 3]
```diff
@@ -7,21 +7,64 @@ date: "2019-11-15"
...
+Venue identifiers are assigned by the Anthology Directory in consultation with submitters.
+Typically, an acronym will be selected ("lrec" for [LREC](http://lrec.org), "acl" for [ACL](http://www.aclweb.org/), etc.).
+
+## Paper numbering
+
+A key notion in file naming is the paper number.
+The general rule is that papers will be numbered consecutively within the bound volume in which they appear.
+When a proceedings is divided into multiple volumes, paper numbers will begin from number '1', with each new volume.
+
+By convention, any front matter is given the paper number '0', padded to the ACL ID length (e.g., 2020.acl-srw.0, 2020.emnlp-1.0).
```
[the remark this answers is not in the document]
Comment (mjpost) [reply hugo/content/info/ids.md]: Thanks for catching that!

File: bin/anthology/bibtex.py (hunk 1 above)
Comment (mjpost) [reply bin/anthology/bibtex.py]: I removed some old logic, but only partially. I'll complete it.

**Pull request merged**